subgraphs/uniswap-forks/src/price/price.ts

```diff
@@ -111,6 +112,18 @@ export function findUSDPricePerToken(
           largestLiquidityWhitelistTokens = whitelistTokenLocked;
           // token1 per our token * nativeToken per token1
 
+          if (NetworkConfigs.getProtocolSlug() == "camelot-v2") {
+            const newPrice = findStablePairUSDPriceForToken(
```

Comment:
So if the `newPrice` does not exist, it needs to retrieve the price the normal way? Or is this always supposed to get the price.

Reply:
Yeah, the `findStablePairUSDPriceForToken` function only returns a price if the pool is marked as a stable (Curve) pool. For other pools the normal uniswap price calculation is used.